sdk/core/Cargo.toml

```diff
@@ -31,14 +31,18 @@ bytes = "1.0"
 hyper-rustls = { version = "0.22", optional = true }
 failure = "0.1"
 async-trait = "0.1"
+futures-io = "0.3"
 oauth2 = "=4.0.0-beta.1"
-reqwest = { version = "0.11", optional = true }
+reqwest = { version = "0.11", features = ["stream"], optional = true }
 paste = "1.0"
 md5 = "0.7"
+rand = "0.8"
```

Comment:
This is for `ExponentialRetryPolicy`?  So it can have a random delay between retries? Are we sure we want it to always be a core dependency?

Reply:
Yes exactly I modeled the retry based on this: [https://docs.microsoft.com/en-us/azure/architecture/best-practices/retry-service-specific#retry-mechanism-3](https://docs.microsoft.com/en-us/azure/architecture/best-practices/retry-service-specific#retry-mechanism-3). BTW rand 0.8 broke wasm so I downgraded to 0.7. Not sure what's different though.